src/Aspire.Dashboard/Components/Controls/ExceptionDetails.razor

```diff
@@ -30,6 +29,6 @@
 
     private void OnExceptionDetailsClicked(MouseEventArgs e)
     {
-        _ = TextVisualizerDialog.OpenDialogAsync(ViewportInformation, DialogService, Loc[nameof(ControlsStrings.ExceptionDetailsTitle)], ExceptionText);
+        _ = TextVisualizerDialog.OpenDialogAsync(ViewportInformation, DialogService, DialogsLoc[nameof(Dashboard.Resources.Dialogs.DialogCloseButtonText)], Loc[nameof(ControlsStrings.ExceptionDetailsTitle)], ExceptionText);
```

Comment:
Pass the DialogsLoc into the method and get the text from it inside the method. Removes some duplication.